README.md

```diff
@@ -41,39 +41,41 @@ export TF_VAR_ASPIRE_AWS_ACCESS_KEY_ID=YOUR_AWS_ACCESS_KEY
 export TF_VAR_ASPIRE_AWS_SECRET_ACCESS_KEY=YOUR_AWS_SECRET_KEY
 export TF_VAR_GIT_PAT=GIT_PERSONAL_ACCESS_TOKEN
 export TF_VAR_ASPIRE_CLUSTER=ASPIRE_CLUSTER_TO_RUN_FOR
-export AWS_ACCESS_KEY_ID=TERRAFORM_AWS_ACCESS_KEY
-export AWS_SECRET_ACCESS_KEY=TERRAFORM_AWS_SECRET_KEY
+export AWS_ACCESS_KEY_ID="$TF_VAR_ASPIRE_AWS_ACCESS_KEY_ID"
+export AWS_SECRET_ACCESS_KEY="$TF_VAR_ASPIRE_AWS_SECRET_ACCESS_KEY"
 ```
 
-3. Navigate to the directory in this repository for the region to use for testing (likely `us-east`).
+3. Load the environment variables from the file with the following command:
 
-4. Initialize Terraform with the following command, replacing `{ ASPIRE_CLUSTER }` with the name of the backend being used for testing (this should match the `TF_VAR_ASPIRE_CLUSTER` variable set above):
-
-```
-terraform init -backend-config="key={ ASPIRE_CLUSTER }/terraform.tfstate"
+```shell
+source .env.terraform
 ```
-e.g.
+
+4. Navigate to the directory in this repository for the region to use for testing (likely `us-east`).
+
+5. Initialize Terraform with the following command:
+
 ```
-terraform init -backend-config="key=aspirestaging/terraform.tfstate"
+terraform init -backend-config="key=$TF_VAR_ASPIRE_CLUSTER/terraform.tfstate"
 ```
 
-5. (Optional but recommended) Verify the actions Terraform will take when applied with the following command:
+6. (Optional but recommended) Verify the actions Terraform will take when applied with the following command:
 
 ```
 terraform plan
 ```
 
-6. Run the following command to apply the changes from Step 5 and provision the specified resources from AWS:
+7. Run the following command to apply the changes from Step 5 and provision the specified resources from AWS:
```

Comment:
"...from Step 5..." should now be "...from Step 6..." since the step numbers have been updated.

Reply:
Good catch. By the way, in the files tab if you press `cmd+g`, you can suggest a code change that we can then commit from the github web UI.